Libraries/Opc.Ua.Client/TraceableSession.cs

```diff
@@ -1987,7 +1990,7 @@ public bool ResendData(IEnumerable<Subscription> subscriptions, out IList<Servic
         /// <inheritdoc/>
         public async Task<(bool, IList<ServiceResult>)> ResendDataAsync(IEnumerable<Subscription> subscriptions, CancellationToken ct = default)
         {
-            using (Activity activity = ActivitySource.StartActivity(nameof(ResendDataAsync)))
+            using (Activity activity = _activitySourceAdapter.StartActivity(nameof(ResendDataAsync)))
```

Comment:
no need to use nameof() here, the adapter does this using [CallingMemberName]

Reply:
Changed the StartActivity signature to not check for callingfunction name as that can add overhead on call for each service routine

Reply:
there won't be any overhead caused by this. CallerMemberName is resolved at compile time, see: https://stackoverflow.com/a/22580624

Reply:
Got it. updated the file to leverage CallerMemberName